eng/pipelines/templates/steps/publish-coverage.yml

```diff
@@ -0,0 +1,12 @@
+parameters:
+  RunCoverage: false
+
+steps:
+
+  - task: PublishCodeCoverageResults@1
```

Comment:
What's generating the actual report file? We disabled coverlet currently but I'm about to re-enable it.